kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/channel/Channel.java

```diff
@@ -339,17 +280,6 @@ public void setEnabled(boolean isEnabled) {
         this.isEnabled = isEnabled;
     }
 
-    /**
-     * Specifies the scale to be applied to the channel value
-     *
-     * @param scale
-     *            a double value that specifies the scale to be applied to the channel value
-     * @since 2.3
-     */
-    public void setScale(double scale) {
-        this.valueScale = scale;
-    }
-
```

Comment:
I changed my mind, since the class is not immutable a client could change first the scale and offset and then the value type and the scale/offset values. So I only restored methods and deprecated them.